src/autohost.ts

```diff
@@ -325,6 +326,43 @@ export class Autohost implements TachyonAutohost {
 			);
 		}
 	}
+
+	shutdown(): void {
+		// Force shutdown (i.e. pressing CTRL+C, CTRL+C in sequence), kill all games then exit.
+		if (this.shuttingDown) {
+			this.logger.warn(
+				'second shutdown signal recieved, forcing shutdown by killing all games and exiting',
+			);
+			this.gamesMgr.killAllGames();
+			process.exit(0);
+		}
+
+		this.shuttingDown = true;
+
+		// No games running, exit immediately.
+		if (this.gamesMgr.gameCount === 0) {
```

Comment:
All those operations in game manager should be done in game manager, so we don't need to expose a bunch of more properties. Effectively: we want to be able to cleanly shut down game manager, that logic needs to be there, just like when we shut down EngineRunner, that logic is embedded there. Then e.g. game manager can emit `close` event when it's done.